1/mdav2/frequencies.py

```diff
@@ -0,0 +1,91 @@
+import sys
+import numpy as np
+sys.path.append('../../extra-files')
+sys.path.append('../../0/mdav2')
+import molecule
+import math
+
+ccms = 3E10 #speed of light in centimeters per second
```

Comment:
Your calculated frequencies are slightly off. It's likely due to the low number of significant figures in these numbers.